shared/common-adapters/modal2.tsx

```diff
@@ -0,0 +1,276 @@
+import * as React from 'react'
+import * as Styles from '../styles'
+import {LayoutChangeEvent} from 'react-native'
+import ScrollView from './scroll-view'
+import {Box2, Box} from './box'
+import BoxGrow from './box-grow'
+import Text from './text'
+import {useTimeout} from './use-timers'
+
+const Kb = {
```

Comment:
Basically a copy of Modal but without the inner popup dialog, which is moved into the custom navigator